### DISCUSSION TITLE
New to Open Source

### DISCUSSION THREAD

**Discussion by GouravSardana** [IndianOpenSourceFoundation/dynamic-cli#155] [General]
Hey @AyuBisht @kushshiv ,

Welcome to the open source world. Thanks for choosing iosf. As per #152 @AyuBisht told about what are the warning. Lemme explain:

1. We followed pep8 styling for our project. You can refer [here](https://www.python.org/dev/peps/pep-0008/)
2. We use Sonar Cloud to review every PR before we check. It'll do some analysis around code quality, styling, extra spaces, unused variables, security and code smell.
3. With of all these check we have some other checks in our GitHub action to see some of the formatting and code quality.

Feel free to use this for open end discussion and Welcome to the community 💯

**Comment by kushshiv** [comment]
@GouravSardana... Thanks for sharing... Really excited to join... 🤝

**Comment by AyuBisht** [comment]
Thank you very much @GouravSardana. 
Thrilled to be a part of the community.

**Comment by GouravSardana** [comment]
Exited to see our community is growing 🎉

**Comment by AyuBisht** [comment]
hey @GouravSardana is there any discord group or something for Dynamic CLI for gssoc22

**Comment by GouravSardana** [comment]
Yes @AyuBisht
